### PR TITLE
Add basic result selection functionality (fixes #12).

### LINKED ISSUE #12: Allow moving selection between results
Currently the top result is always selected, and it is not possible to pick or navigate to the other results.

Users should be able to navigate the result list with the arrow keys.

### PR DESCRIPTION
**Pull request by Cheddam**
Also includes updates to JS for linting conformity and a tweak in placement of the global keyboard shortcut.

### CODE REVIEW

File: js/switchflit.js [hunk 1]
```diff
@@ -1,60 +1,73 @@
...
+      }
+    },
+    shiftUp() {
+      if (this.selectedResult > 0) {
+        this.selectedResult = this.selectedResult - 1;
+      }
+    },
+
+    openResult() {
+      if (this.filteredRecords.length < 1) return true;
```
Comment (edlinklater) [inline js/switchflit.js]: This looks like a failure scenario, but is returning `true`?

File: js/switchflit.js [hunk 2]
```diff
@@ -1,60 +1,73 @@
...
+
+      window.location = this.filteredRecords[this.selectedResult].link;
+      return false;
```
Comment (edlinklater) [inline js/switchflit.js]: Is `return` necessary here? If so, should it be false?

File: js/switchflit.js [hunk 3]
```diff
@@ -1,60 +1,73 @@
...
+    },
+  },
+
+  created() {
+    fetch(`/switchflit/${this.dataobject}/records`)
+      .then(response => response.json())
+      .then((records) => {
```
Comment (edlinklater) [inline js/switchflit.js:54]: Is there some syntactical reason `((records))` is in double brackets but `(response)` is single?

File: js/switchflit.js [hunk 4]
```diff
@@ -1,60 +1,73 @@
...
+
+        this.fuse = new Fuse(this.records, { keys: ['title'] });
+      });
+  },
+
+  ready() {
+    document.addEventListener('keydown', (event) => {
+      if (event.keyCode === 27) {
+        this.hide();
+      } else if ((navigator.platform === 'MacIntel' && event.metaKey && event.keyCode === 75) || (event.ctrlKey && event.keyCode === 75)) {
```
Comment (edlinklater) [inline js/switchflit.js]: Would this work on e.g. an iPad? If so, might need another platform other than just MacIntel.

File: js/switchflit.js (hunk 1 above)
Comment (Cheddam) [reply js/switchflit.js]: Yep, didn't really think this one through - the returned value has no impact on the outcome, but it should be more sensible. Will fix.

File: js/switchflit.js (hunk 2 above)
Comment (Cheddam) [reply js/switchflit.js]: Again, the returned value has no impact (other than possibly stopping the handled event from propagating) but this is extraneous code. Will fix.

File: js/switchflit.js (hunk 3 above)
Comment (Cheddam) [reply js/switchflit.js:54]: Extraneous, not picked up by linter. Will fix.

File: js/switchflit.js (hunk 4 above)
Comment (Cheddam) [reply js/switchflit.js]: Solid point, investigating...

File: js/switchflit.js (hunk 3 above)
Comment (Cheddam) [reply js/switchflit.js:54]: Cancel that, linter wants this in place. ¯_(ツ)_/¯

**Review by Cheddam — COMMENTED** [review]
Changes made, please re-review.

**Pull request merged**